src/Server/Processors/AeTitleJobProcessor.cs

```diff
@@ -208,9 +209,15 @@ private void OnTimedEvent(Object source, System.Timers.ElapsedEventArgs e)
                         }
                         else
                         {
-                            _ = _jobs.TryAdd(_instances[key]);
-                            _instances.Remove(key);
-                            _logger.Log(LogLevel.Information, $"Timeout elapsed waiting for {_grouping} {key}");
+                            if (_jobs.TryAdd(_instances[key]))
+                            {
+                                _instances.Remove(key);
+                                _logger.Log(LogLevel.Information, $"Timeout elapsed waiting for {_grouping} {key}");
+                            }
+                            else
+                            {
+                                _logger.Log(LogLevel.Error, $"Error while queuing new instance collection with key={key}, grouping={_grouping}.");
```

Comment:
Could the message a little more specific, like "Failed to enqueue ...."?

Reply:
Updated